AuthenticationExample/AuthenticationExample/AuthenticationApp.swift

```diff
@@ -18,19 +18,9 @@ import ArcGIS
 
 @main
 struct AuthenticationApp: App {
-    @ObservedObject var authenticator: Authenticator
```

Comment:
`AuthenticationApp` owns the Authenticator instance so this should be a `StateObject` as noted [here](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/pull/392#discussion_r1295041825).